examples/get_iss_breast_data.py

```diff
@@ -23,6 +23,15 @@ def __init__(self, file_path):
     def shape(self) -> Tuple[int, ...]:
         return SHAPE
 
+    @property
+    def coordinates(self) -> Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]:
```

Comment:
Funny, Matt Cai mentioned today that coordinates are the one thing that's not in the filename when delivering single-plane TIFFs. Do we have a plan on how users should provide these by default? 

... a json file mapping filenames to coordinates, perhaps? It'll be important to set up such a default soon.

Reply:
> Do we have a plan on how users should provide these by default?

I don't envision that we'll be strongly opinionated about this process.